File: PizzaShopAdmin/ClientApp/src/dto/User/UserDto.ts
import { UserRole } from './Enum/UserRole';

export class UserDto {
  public id: number;
  public username: string;
  public name: string;
  public surname: string;
  public birthday: Date;
  public phone: string;
  public address: string;
  public email: string;
  public role: UserRole;
  public CreatedAt: Date;
}

File: PizzaShopAdmin/Dto/User/UserDto.cs
using PizzaShop.EntityFramework.Entities.Enum;
using System;

namespace PizzaShopAdmin.Dto.User
{
    public class UserDto
    {
        public int Id { get; set; }
        public string Username { get; set; }
        public string Name { get; set; }
        public string Surname { get; set; }
        public DateTime Birthday { get; set; }
        public string Phone { get; set; }
        public string Address { get; set; }
        public string Email { get; set; }
        public Role Role { get; set; }
        public DateTime CreatedAt { get; set; }
    }
}
